### PR TITLE
Merge 1.1 changes from main to 1.1

### PR DESCRIPTION
**Pull request by lezzago**
### Description
* Add Release Notes
* Merge in changes from main
 
### Issues Resolved
N/A
 
### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by qreshi — APPROVED** [review]

File: .github/workflows/links.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+name: Link Checker
+on:
+  push:
+    branches: [ main ]
```
Comment (dbbaughe) [inline .github/workflows/links.yml:4]: Do you want to update these and add "1.*" so it runs on all cut 1.y specific branches and the "1.x"?

File: ADMINS.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Admins
+
+| Admin           | GitHub ID                               | Affiliation |
+| --------------- | --------------------------------------- | ----------- |
+| Henri Yandell   | [hyandell](https://github.com/hyandell) | Amazon      |
+
+[This document](https://github.com/opensearch-project/.github/blob/main/ADMINS.md) explains what admins do in this repo. and how they should be doing it. If you're interested in becoming a maintainer, see [MAINTAINERS](MAINTAINERS.md). If you're interested in contributing, see [CONTRIBUTING](CONTRIBUTING.md).
```
Comment (dbbaughe) [inline ADMINS.md:7]: Does it explain what they do? 🤔

File: .github/workflows/links.yml (hunk 1 above)
Comment (lezzago) [reply .github/workflows/links.yml]: Updated this

File: ADMINS.md (hunk 2 above)
Comment (lezzago) [reply ADMINS.md:7]: Yup: `As an admin you own stewartship of the repository and its settings. Admins have admin-level permissions on a repository. Use those privileges to serve the community and protect the repository as follows.`

**Review by dbbaughe — APPROVED** [review]

**Pull request merged**